### ISSUE TITLE
Level: Double Showdown

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#496]
```
#######
Q      
   ( Oy
#######
Q      
   ( Oy
#######
:dig=2
:name=Double Showdown
:num_rabbits=2
:num_to_save=2
:solution.1=dig&(3,2);(3,5);until:WON
:solution.2=dig&(4,2);(3,5);until:WON
```

**Comment by GamingInky** [comment]
This level is possible and very interesting.

**Comment by andybalaam** [comment]
I don't like this yet. Needs more work.

**Comment by colonelfazackerley** [comment]
dig&(3,2);(3,5);9

This is a solution. looks like a bug, though. the tokens are on bridges, but then the rabbots that pick them up dig under the bridge.

**Comment by andybalaam** [comment]
That is the intended solution, and the weirdness of that is one of the reasons I don't like it

**Comment by tttppp** [comment]
My solution is:
```dig;;(2,2);10```

This is weird because the rabbot kindly waits for the rabbit to finish digging.

**Comment by tttppp** [comment]
As well as colonel's solution there's also a very similar:
```dig&(4,2);(3,5);7```
This drops both rabbots through the bottom.

**Comment by andybalaam** [comment]
I totally meant you to save both - updated now, meaning some of the above solutions are no longer valid.

**Comment by andybalaam** [comment]
Released in 0.11
